data/parse.py

```diff
@@ -133,24 +133,28 @@ def is_nan(cell) -> bool:
             location = a[y, 0]
             vaccinated = a[y, x]
 
-            if "population estimates" in dose.lower():
+            ignore = [
                 # Ignore population estimates.
-                continue
-            if "% who have had at least 1 dose" in dose.lower():
+                "population estimates",
                 # Ignore precalculated %
+                "% who have had at least 1 dose",
+                "% who have had both doses",
+                # Ignore dose summaries.
+                "total 1st doses",
+                "total 2nd doses",
+            ]
+
+            if any(map(lambda d: d in dose.lower(), ignore)):
                 continue
             if type(group) == str and "percent of all" in group.lower():
                 # Ignore percentage reports.
                 continue
-            if dose in ["Total 1st Doses", "Total 2nd Doses"]:
-                # Ignore dose summaries.
-                continue
 
-            is_dose_and_group_all = dose == "Cumulative Total Doses to Date"
+            is_dose_and_group_all = "cumulative total doses to date" in dose.lower()
 
-            if dose == "1st dose":
+            if "1st dose" in dose.lower():
```

Comment:
Remove the substring matching? Unknown column titles that aren't in `ignore` but contain the string `"1st dose"` will lead to unhandled errors in the data. Better crash than have something incorrect

Reply:
For some reason they renamed the fields in this sheet to `1st dose5`. Should I just allow that as a static allowed value instead?